massa-module-cache/src/controller.rs

```diff
@@ -55,16 +58,21 @@ impl ModuleCache {
                 warn!("compilation of module {} failed with: {}", hash, e);
                 ModuleInfo::Invalid
             }
-        }
+        };
+        let end = Instant::now();
+        let elapsed = end.duration_since(start).as_micros();
+        debug!("TIMER: cl compilation took {} μs", elapsed);
```

Comment:
remove

Reply:
You want to remove this one but not the SP compilation timer? Or did you miss it and want both removed?

Reply:
oh :) both if the calibration is done